page_delete_test.go

```diff
@@ -124,8 +124,8 @@ func TestPageDeleteSub(t *testing.T) {
 	})
 
 	go func() {
-		time.Sleep(20 * time.Second)
-		t.Errorf("\n%s", "Stream keep-alive does not reconnect to the server")
+		time.Sleep(10 * time.Second)
```

Comment:
Do we really need to sleep for `10 seconds` ? Can we make the sleep time smaller?

Reply:
it was some kind of timeout. 

It does not work right. It will cause an issue if all tests take more then 10 seconds. I need to update this logic

Reply:
Let's wait group

Reply:
Let's check errors and a final number of messages